modules/gateways/blockonomics/blockonomics.php

```diff
@@ -500,7 +554,9 @@ public function insertOrderToDb($id_order, $blockonomics_currency, $address, $va
                 ]
             );
         } catch (Exception $e) {
-            exit("Unable to insert new order into blockonomics_orders: {$e->getMessage()}");
+            if ($blockonomics_currency != 'usdt') {
```

Comment:
This change looks unnecessary. We should always catch errors inserting an order to the db